### ISSUE TITLE
Template update 02/19

### ISSUE THREAD

**Issue by baytechanson** [calwater/cws-template#1]
1. Any links to calwater.com need to be full paths instead of relative links.
2. Delete the “Español” link in the header, since none of these pages are available in Spanish.
3. Fix the issue with the pagination.
4. Remove unwanted classes.

**Comment by baytechanson** [comment]
@dcancilla @jlien-calwater 
Template updated. Please check and let me know.

**Comment by dcancilla** [comment]
@baytechanson I have two comments:

<ul>
<li>I don't believe the "About Us" link at the top of the page should be yellow.</li>
<li>I'm having some trouble with the menus when I view the page using Firefox. When I point at "Customer Care" (for example) the menu opens, but as I move my mouse down the menu the menu often closes before I reach the bottom. Can you see if you have the same issue? It seems okay in other browsers.</li>
</ul>

**Comment by jlien-calwater** [comment]
@baytechanson there is issue with the modal. It is not showing correctly.

**Comment by baytechanson** [comment]
@dcancilla We can't reproduce the menu issue with Firefox, but we tried to enhanced the coding, please double check. Let me know if it's been fixed.

**Comment by baytechanson** [comment]
@jlien-calwater Some files from bootswatch were missing, added, please check.

**Comment by dcancilla** [comment]
@baytechanson I'm not seeing a difference with the menus, but if you can't reproduce it, it may just be something with my particular setup. Let's let this go for now and see if anyone else has trouble.

**Comment by baytechanson** [comment]
@dcancilla Got it.

**Comment by jlien-calwater** [comment]
@baytechanson I removed the resources and put this inline in the all_components.html file.
bc3d1e4db842a569bc46656683493f1e71213832

This way, we won't have unwanted files lying around when we pass this around.

I also updated the template (f11749ef28c04325a47f7acc03770b5e0dc13f1c) and move the JS toward the body for performance reasons.

Also, part of Start/Stop service uses jQuery UI datepicker. I don't know if you need to edit the look and feel for this to match our current template, but this is the UI it is currently using: [jQuery UI](http://jqueryui.com/themeroller/#!zThemeParams=5d00000100f105000000000000003d8888d844329a8dfe02723de3e570162cf18d27450c454aa2f2e374d1d127fabb9ebc66c3ee5a70fb154870371702db6720bcd83f0eca39ba10d7f6bd6474397674511ae8a0f2315c02102ab95cc2ed73d5845083639df983b144bb7faff3827866eddf2de2c4b4ce10c1c275c51e538a6e77075f58ac23b502242dd2043e740e4dae2871fe39522b6b7b06cfbd674289eb67db5ab4a8c78ec39eb4b28e1d0131a7a7fd20dec8abb289f7d2014380f3fb1e4b4dd19bc33a4aaa4242fb1a3c6013f23c8bc6df387c4c3080e7d63aa3bd0dfae5931c5b38ef8aa14000c1148aac7111b952bed17f0948cb4ac8a81034d80012797841da7ee60522c67f000b972e1da0177b75cfc42f6d9214ca7d523485071c520bbae1eb7ac9e29f54911bfc77e1956adf8d80a0c74e98e1e3ce56923c9046776b94f4af48351a55d2bae278991c21b5e3045f291249de61cec9abef2eb1e24307f75188479d8ee3fc044e5bde99cb97cc359f391b8bb268875fd50f7354ff4d0b3e30502aa2ad22fd1006fa38abec1157f6bccc22da7e4fa3c7eeb26d1f43961c60991abe5fb312f057efe746f1b836578f547d27f8816470762c42e4e5704482e9901416c220f57d3f7ea9fe5b82a04f4eb4dba22443010909ea2855a61cad25430e79d0061efab2b4e60ba1b8f6a4ef4deaea89fecfbe69224f6e520117043211bc0cb85a6984035d279999ea03150c7a7e484d316da91da6ad4f4bfd80aba6)

I updated jQuery to the latest: 4e2d2e54198419d3a6a86ba1afa34d97acaa80ba

**Comment by jlien-calwater** [comment]
@dcancilla check out all_components.html and see if you are happen with it. To me it looks good.

**Comment by dcancilla** [comment]
@jlien-calwater I agree that the all_components.html page looks good, but it doesn't have most of our design elements. What is that page going to be used for?

**Comment by jlien-calwater** [comment]
@dcancilla these are all components that can be used when developing application. It is just to show what we have and how does it look/work.

**Comment by dcancilla** [comment]
@jlien-calwater Okay, I get it. In that case, I think it looks fine. We might have to make adjustments in the future if a unique design situation arises, though.

**Comment by jlien-calwater** [comment]
Form radio/checkbox, the label is not on the same line as the button.

![image](https://cloud.githubusercontent.com/assets/3473846/13362817/2752d6c0-dc7b-11e5-89fe-2c1eadc958fc.png)

**Comment by jlien-calwater** [comment]
@dcancilla I updated the thank you page.

Old:
![image](https://cloud.githubusercontent.com/assets/3473846/13368947/6bdf82a6-dca4-11e5-99e5-701f94f191ba.png)


New:
![image](https://cloud.githubusercontent.com/assets/3473846/13368960/959d8d04-dca4-11e5-991e-bdd0c0ff0aef.png)

And responsive:
![image](https://cloud.githubusercontent.com/assets/3473846/13368966/a464805e-dca4-11e5-836d-1b25a9cfac6d.png)

**Comment by dcancilla** [comment]
@jlien-calwater Very nice! The only suggestion I have is that it might be nice if there were a little less leading in the text below the light bulb.

**Comment by baytechanson** [comment]
@jlien-calwater The alignment of radio/checkbox and the text has been fixed.

**Comment by jlien-calwater** [comment]
@dcancilla the leading is dynamic I believe based on the percentage of your width. So it isn't going to be 100% perfect all the time.

You should be able to POST now.

I just need to investigate a server error. I think it errors due to a timeout period with the server settings. I will let you know, but you can test now.

@baytechanson thanks, I will merge this with what I have.

**Comment by jlien-calwater** [comment]
@baytechanson 

Found a resource missing.

![image](https://cloud.githubusercontent.com/assets/3473846/13435904/74077606-df91-11e5-8b50-bc4b9791071f.png)

There are others:

nav-point.png
images/search-icon.png

Please double check the custom.css file and make sure the resource are there and in their correct folder.

**Comment by jlien-calwater** [comment]
The navigation and body have a different z-index renders unwanted result.

![image](https://cloud.githubusercontent.com/assets/3473846/13445976/98a2ff0c-dfc4-11e5-886a-56e61d36dae3.png)

The logo, background curve SHOULD be in the background. The navigation POP-UP menu SHOULD be in the foreground.

@baytechanson please let me know if this makes any sense to you. I am hoping that whatever you applying here you are making changes to the WordPress template as well.

**Comment by baytechanson** [comment]
@jlien-calwater 
Removed the style related with "dotted-line.png" and "nav-point.png".
"search-icon.png" is existing and working fine.

**Comment by baytechanson** [comment]
@jlien-calwater We have removed the "z-index" attribute for the header. Please check and let me know.
BTW, we have updated the wordpress template with the similiar changes.

**Comment by jlien-calwater** [comment]
@baytechanson there's are a bunch of .png in the custom css. Could you confirmed that you have checked all of them?

**Comment by baytechanson** [comment]
@jlien-calwater There are 12 .png left in the custom css, and they are all in use.
But we have deleted some other image files: 02371d08358dd0331c7b75023380a064bdd74448

**Comment by jlien-calwater** [comment]
@dcancilla I noticed that we are using [Raleway](https://www.google.com/fonts/specimen/Raleway) for our font.

Do you know the numbers do not line up?

![image](https://cloud.githubusercontent.com/assets/3473846/13534934/639177ba-e1ec-11e5-8f4b-c6c49d4869b9.png)

They look pretty weird in a input box.

![image](https://cloud.githubusercontent.com/assets/3473846/13535005/c24b0046-e1ec-11e5-8e0a-3628da79f349.png)

**Comment by dcancilla** [comment]
@jlien-calwater Raleway has numeric descenders, so believe it or not, that's correct. I can't say I necessarilly agree with the design decision, but it's not technically wrong.

**Comment by RobatBaytech** [comment]
@dcancilla @jlien-calwater 

How is the progress on template review? I guess it's getting closer to fully refined. And I'd also like to ask when can we launch the website will new design. If you can give me an estimate date, that will be terrific. This can help me manage my team's schedule.

Thanks,
